### PR TITLE
[Merged by Bors] - chore: ensure there is a replacement identifier or text suggestion for all deprecations

### PR DESCRIPTION
**Pull request by kim-em**
This is for compatibility with the stricter requirements of leanprover/lean4#6112.

Anyone is welcome to replace the "No deprecation message was provided." messages in other PRs! :-)

### CODE REVIEW

File: Mathlib/MeasureTheory/Function/L1Space.lean [hunk 1]
```diff
@@ -443,7 +443,7 @@ lemma Integrable.of_finite [Finite α] [MeasurableSingletonClass α] [IsFiniteMe
 
 /-- This lemma is a special case of `Integrable.of_finite`. -/
 -- Eternal deprecation for discoverability, don't remove
-@[deprecated Integrable.of_finite, nolint deprecatedNoSince]
+@[deprecated Integrable.of_finite (since := "2024-10-05"), nolint deprecatedNoSince]
```
Comment (sgouezel) [inline Mathlib/MeasureTheory/Function/L1Space.lean:446]: I guess the `nolint` doesn't do anything here? Is it used somewhere else in the library?

**Review by sgouezel — COMMENTED** [review]
bors d+
Thanks!

File: Mathlib/MeasureTheory/Function/L1Space.lean (hunk 1 above)
Comment (acmepjz) [reply Mathlib/MeasureTheory/Function/L1Space.lean:446]: I guess no `since` is intentional, as it never expires and hence is searchable by e.g. Loogle.

Comment (kim-em) [reply Mathlib/MeasureTheory/Function/L1Space.lean:446]: I think we'll just remove the `deprecatedNoSince` linter shortly.

**Comment by kim-em** [comment]
bors merge

**Pull request closed without merging**